model/src/test/java/org/projectnessie/model/TestContentKey.java

```diff
@@ -260,6 +260,7 @@ void doubleByte() {
   @Test
   void blockZeroByteUsage() {
     assertThrows(IllegalArgumentException.class, () -> ContentKey.of("\u0000"));
+    assertThrows(IllegalArgumentException.class, () -> ContentKey.of("%00"));
```

Comment:
(same comment as in ContenbKey) IIRC the `%` char should already be forbidden. Maybe deserves a separate test?